exercises/using-classes/practice/src/main/java/translationworkflow/TranslationWorkflowImpl.java

```diff
@@ -11,39 +11,43 @@
 
 public class TranslationWorkflowImpl implements TranslationWorkflow {
 
-  ActivityOptions options =
-      ActivityOptions.newBuilder().setStartToCloseTimeout(Duration.ofSeconds(5)).build();
+  private final ActivityOptions options =
+      ActivityOptions.newBuilder()
+          .setStartToCloseTimeout(Duration.ofSeconds(5))
+          .build();
 
   private final TranslationActivities activities =
       Workflow.newActivityStub(TranslationActivities.class, options);
 
   @Override
+  // TODO Update the method to take a single input of type TranslationWorkflowInput 
+  // as a parameter and to return a value of type TranslationWorkflowOutput
```

Comment:
Why do we have this TODO here when the work the TODO is saying to do has already been done below?